server/app/views/ViewUtils.java

```diff
@@ -75,10 +73,8 @@ public ScriptTag makeWebJarsTag(String assetsRoute) {
   /**
    * Generates an HTML link tag for loading the CSS file found at public/stylesheets/[filename].css.
```

Comment:
Should the path in this comment be `public/dist/[filename].css`? I think you said "no" when we discussed offline, but I didn't understand why.

Reply:
You're right, I do need to change the path in that comment.  It should say "found at public/main/[path].css".  Here are the two paths it will receive as an argument:
- `stylesheets/tailwind`
- `dist/uswds.min`

`tailwind.css` ends up under` public/main/stylesheets` because when SBT builds it copies everything from these assets directories directly into `public/main`.
![image](https://github.com/civiform/civiform/assets/134556723/8d3eec22-7a9b-4b0f-8a0b-19c9b2728737)

`uswds.min.css` end up under `public/main/dist` because it was bundled with Webpack and that is the output folder we specified.